.github/workflows/presubmit.yml

```diff
@@ -26,6 +26,14 @@ jobs:
     - uses: actions/checkout@v3
     - name: Build go images
       run: make bin
+  conform:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
nit: is this necessary?

Reply:
Yes, this is necessary if we want to allow an arbitrary number of commits per PR (I think we do). Otherwise, it errors out because it can't see the history.